### DISCUSSION TITLE
Is it possible to put group::Column inside of group::Scroll?

### DISCUSSION THREAD

**Discussion by LaBatata101** [fltk-rs/fltk-rs#907] [Q&A]
I tried here with this code but it doesn't seems to work.
```rust
fn main() {
    let app = app::App::default().with_scheme(app::Scheme::Gtk);
    let theme = ColorTheme::from_colormap(color_themes::BLACK_THEME);
    theme.apply();

    let mut win = window::Window::default()
        .with_size(800, 600)
        .with_label("test column");

    let mut scroll = group::Scroll::default().size_of_parent();
    let mut column = group::Column::default_fill();

    for i in 0..56 {
        let mut frame = Frame::default().with_label(&i.to_string()).with_align(Align::Inside | Align::Left);
        frame.set_frame(FrameType::FlatBox);
        frame.set_size(frame.label_size(), frame.label_size());
    }
    column.end();

    scroll.end();

    win.make_resizable(true);
    win.end();
    win.show();

    app.run().unwrap();
}
```

**Comment by MoAlyousef** [comment]
Can you be more specific?
To get the scroll to work as a scroll, you would have to give the column a longer size than the scroll.

**Reply by LaBatata101** [reply]
Here's a screenshot using the group::Column inside of group::Scroll with 56 Frame, as you can see the scrollbar doesn't appear and it looks like group::Column is resizing it's children to fit the window.
![image](https://user-images.githubusercontent.com/20308796/134780457-66daaaec-2b2c-4860-824a-8174e319a953.png)

Here I'm using group::Pack inside of group::Scroll with the same 56 Frame, in this case the scrollbar works.
![image](https://user-images.githubusercontent.com/20308796/134780501-ddc4efa8-34ff-4c28-8269-a1e3c464c929.png)

**Reply by MoAlyousef** [reply]
A Pack has the property of expanding automatically. To get the same effect with a Column (which uses a Flex internally), you would have to give it a longer size than the scroll.
If you try to call auto_layout() on the pack after pack.end(), it will give you the same effect as the Column.

**Reply by LaBatata101** [reply]
Thanks!
